resources/resources/software/aws/toolkits/resources/localized_messages.properties

```diff
@@ -280,7 +280,7 @@ sam.build.title=SAM build
 sam.build.running=Running SAM build
 sam.build.failed=SAM build command failed
 sam.build.succeeded=SAM build command succeeded
-cloudformation.stack.delete.action=Delete CloudFormation Stack
+cloudformation.stack.delete.action=Delete Stack
```

Comment:
Please add ellipses (`Delete Stack...`) to indicate that further prompting/interaction will happen after selecting this item.

Reply:
Good catch